arduino/smartcar/smartcar.ino

```diff
@@ -14,18 +22,39 @@ void setup() {
 }
 
 void loop() {
-  car.setSpeed(90);
+  car.setSpeed(70); 
   preventCrash();
 }
 
-void preventCrash() {
-  unsigned int fDistance = front.getDistance();
-  if (fDistance > 0 && fDistance < 200) {
+void preventCrash(){
+  preventCrashWithUltrasonic(); 
+  preventCrashWithInfrared(frontIR, 0, 40); // Helps in short distances object detection.
+}
+
+void preventCrashWithUltrasonic() {
+  unsigned int distanceUS = frontUS.getDistance(); // min distance: 0.2 m, MAX distance: 4 m.
+  if (distanceUS > 0 && distanceUS < 20) {
```

Comment:
You changed the wrong value. Check your signs!

Reply:
Fixed